apis/keda/v1alpha1/scaledobject_types.go

```diff
@@ -254,21 +260,49 @@ func (so *ScaledObject) GetHPAMaxReplicas() int32 {
 	return defaultHPAMaxReplicas
 }
 
+// GetDefaultHPAMaxReplicas returns defaultHPAMaxReplicas
+func (so *ScaledObject) GetDefaultHPAMaxReplicas() int32 {
+	return defaultHPAMaxReplicas
+}
+
+// CheckReplicasNotNegative checks that Min/Max ReplicaCount defined in ScaledObject are not negative
+func (so *ScaledObject) CheckReplicasNotNegative(replicas ...int32) bool {
+	for _, replica := range replicas {
+		if replica < 0 {
+			return false
+		}
+	}
+	return true
+}
+
+// GetIdleReplicasIfDefined returns bool based on whether idleRelicas is defined
+func (so *ScaledObject) GetIdleReplicasIfDefined() bool {
+	return so.Spec.IdleReplicaCount != nil
```

Comment:
imho, the check is more readable than `GetIdleReplicasIfDefined` name, I'd ditch this helper too and keep `so.Spec.IdleReplicaCount != nil` which is very common convention in kubernetes codebase.

Reply:
> imho, the check is more readable than `GetIdleReplicasIfDefined` name, I'd ditch this helper too and keep `so.Spec.IdleReplicaCount != nil` which is very common convention in kubernetes codebase.

ok, I will follow the code style you suggest.